content/en/docs/application-store/built-in-apps/radondb-postgresql-app.md

```diff
@@ -0,0 +1,84 @@
+
+   ![deploy-RadonDB PostgreSQL](/images/docs/appstore/built-in-apps/radondb-postgresql-app/deploy-radondb-postgresql.png)
+
+3. Set a name and select an app version. Make sure RadonDB PostgreSQL is deployed in `demo-project` and click **Next**.
+
+   ![confirm-deployment](/images/docs/appstore/built-in-apps/radondb-postgresql-app/confirm-deployment.png)
+
+4. In **App Configurations**, you can use the default configuration or customize the configuration by editing the YAML file directly. When you finish, click **Deploy**.
+
+   ![set-app-configuration](/images/docs/appstore/built-in-apps/radondb-postgresql-app/set-app-configuration.png)
+
+5. Wait until RadonDB PostgreSQL is up and running.
+
+   ![RadonDB PostgreSQL-running](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-running.png)
+
+### Step 2: View PostgreSQL Cluster status
+
+1. On the **Overview** page of the project `demo-project`, you can see a list of resource usage in the current project.
+
+   ![project-overview](/images/docs/zh-cn/appstore/built-in-apps/radondb-postgresql-app/project-overview.png)
+
+2. In **Services** under **Application Workloads**, click the Service name of RadonDB PostgreSQL.
+
+   ![RadonDB PostgreSQL-service](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-service.png)
+
+3. In **Workloads** under **Application Workloads**, click the **StatefulSets** tab and you can see the StatefulSets are up and running.
+
+   ![statefulsets-running](/images/docs/zh-cn/appstore/built-in-apps/radondb-postgresql-app/statefulsets-running.png)
+
+   Click a single StatefulSet to go to its detail page. You can see the metrics in line charts over a period of time under the **Monitoring** tab.
+
+   ![statefulset-monitoring](/images/docs/zh-cn/appstore/built-in-apps/radondb-postgresql-app/statefulset-monitoring.png)
+
+4. In **Pods** under **Application Workloads**, you can see all the Pods are up and running.
+
+   ![pods-running](/images/docs/zh-cn/appstore/built-in-apps/radondb-postgresql-app/pods-running.png)
+
+5. In **Volumes** under **Storage**, you can see the PostgreSQL Cluster components are using persistent volumes.
+
+   ![volumes](/images/docs/zh-cn/appstore/built-in-apps/radondb-postgresql-app/volumes.png)
+
+   Volume usage is also monitored. Click a volume item to go to its detail page. Here is an example of one of the data nodes.
+
+   ![volume-status](/images/docs/zh-cn/appstore/built-in-apps/radondb-postgresql-app/volume-status.png)
+
+### Step 3: Access RadonDB PostgreSQL
+
+1. On the **Overview** page of the project `demo-project`.
+
+2. Under **Pods**, expand the menu to see container details, and then click the **Terminal** icon.
+
+   ![RadonDB PostgreSQL-terminal](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-terminal.png)
+
+3. In the pop-up window, enter commands in the terminal directly to use the app.
```

Comment:
`the pop-up window` -> `the displayed dialog box`

Reply:
done